src/vaski-hop.js

```diff
@@ -58,72 +127,48 @@ function initializeVariables() {
 	allComputerImgs = [scissors2, paper2, well2];
 };
 
-
-var highlightUserSelection = function (htmlObject) {
-	removeAllUserSelections();
-	htmlObject.classList.add('active');
-};
-
 function addEventListeners() {
-	scissors.addEventListener("click", function () {
-		highlightUserSelection(scissors);
-		playerSelection = typeScissors;
-		showPlayerSelection();
-	});
-
-	paper.addEventListener("click", function () {
-		highlightUserSelection(paper);
-		playerSelection = typePaper;
-		showPlayerSelection();
-	});
-
-	well.addEventListener("click", function () {
-		highlightUserSelection(well);
-		playerSelection = typeWell;
-		showPlayerSelection();
-	});
-
-
-	//	setBorder+removeBorder
-	// setPlayerSelection
 
+	for (let i = 0; i < allSelectionTypes.length; i++) {
 
-	var nextTurn = function () {
-		continueButton.addEventListener("click", displayComputerSelection)
+		allPlayerImageSelectors[i].addEventListener("click", function () {
```

Comment:
šitam bloke naudoji skirtingo abstrakcijos lygio sąvokas:
highlightUserSelection - kalba apie UI koncepciją, kaip UI matai per žaidimo prizmę.
removeErrorStyle() - visiškai kito lygmens sąvoka - ji kalba apie CSS stilių, ir šitam lygmenyje egzistuoja DOM, html, css o ne žaidimo sąvokos.
**Vienoje funkcijoje naudok vieno lygmens sąvokas.**
Konkrečiai tai reikštų: pvz sukuri naują funkciją kurios pavadinimas yra visai kito lygmens sąvokos, o jos viduje iškvieti šitą žemesnio lygio funkciją.